drivers/iscsilib.py

```diff
@@ -378,7 +378,7 @@ def get_luns(targetIQN, portal):
 
 
 def is_iscsi_daemon_running():
-    cmd = ["/sbin/pidof", "-s", "/sbin/iscsid"]
+    cmd = ["/sbin/pidof", "-s", "/usr/sbin/iscsid"]
```

Comment:
The systemd way to do this is
```
 systemctl show --property "MainPID" iscsid
```

Reply:
Also changing all the calls below to `service` to be the equivalent `systemctl` commands.